src/main.js

```diff
@@ -1 +1,22 @@
-'use strict';
+// eslint-disable-next-line strict
+function updateClock() {
```

Comment:
The task requires to create a stopwatch with only CSS animations, but the given solution uses JavaScript to update the clock. Please make sure to use CSS animations for this task.

Reply:
delete main.js file